seagence/README.md

```diff
@@ -0,0 +1,40 @@
+# Seagence for Datadog
+
+## Overview
+
+[Seagence][1] is a defect monitoring platform for Java applications that detects defects including root cause in real-time.
```

Comment:
The overview section should include:

1. What Seagence does in 1-2 sentences
2. What will customers get from this integration (This should be clear to mutual customers but also potential new customers)
3. Types of data being sent to Datadog

Reply:
Updated Overview section as suggested.